### PR TITLE
NFTFair intro

### PR DESCRIPTION
**Pull request by Liao1**
done by @xiaofeng @preston

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -44,6 +44,19 @@ nav:
       - Node Types: ./canarynet/node/node-type.md
       - Run Full Node: ./canarynet/node/run-full-node.md
       - Join as Validator: ./canarynet/node/run-validator.md
+  - NFTFair:
+    - Introduction: ./nftfair/introduction.md
+    - User Info:
```
Comment (duxiaofeng-github) [inline mkdocs.yml:49]: user info? user guide?

**Comment by melynx** [comment]
@preston4896 can you update the main menu's left bar to add in the nftfair "banner" + navigation?

File: mkdocs.yml (hunk 1 above)
Comment (Liao1) [reply mkdocs.yml:49]: hhh

File: docs/nftfair/introduction.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Introduction
+Automata NFTFair is A NFT launch platform, designed around principles of fairness and efficacy for a community-led experience is how we would describe NFTFair. Metadata exploits pre-sniping are all predatory practices that make it unfair for the average user to get their hands on a new collection, much less obtain rare or lucrative items.
```
Comment (Liao1) [inline docs/nftfair/introduction.md:2]: [Automata NFTFair](https://nftfair.app) is A NFT launch platform

File: mkdocs.yml (hunk 1 above)
Comment (preston4896) [reply mkdocs.yml:49]: Well technically, the section doesn't really provide any sort of "guide" for the users, but really presents more information about the platform.

**Review by melynx — APPROVED** [review]
LGTM

**Pull request merged**